assets/src/apps/delivery/components/ActivityRenderer.tsx

```diff
@@ -404,13 +402,7 @@ const ActivityRenderer: React.FC<ActivityRendererProps> = ({
           target = ownerActivity ? `${ownerActivity.id}|${target}` : `${target}`;
         }
       }
-      const originalValue = snapshot[target];
-      const typeOfOriginalValue = typeof originalValue;
-      const evaluatedValue =
-        typeOfOriginalValue === 'string' && initStateFacts[key] !== CapiVariableTypes.MATH_EXPR
```

Comment:
pretty sure that the only thing that was added was about the math expr, we needed to parse these to get the values to show up in mission control

Reply:
Hey @bsparks,

Actually we don't need that because we are already doing that in deck.ts when we were loading the fact. We were doing the same thing here so it shouldn't matter..

Also, I have tested the mission control thing, it displays the selected planet text in mission control logs